src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -72,9 +72,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             onSpeechQueued: null
         },
         members: {
-            queue: []
+            queue: [],
+            //
+            currentUtterance: {}
```

Comment:
I think my suggestion of sticking this in the queue is better, since otherwise you have to make extra effort to clean it up when the utterance is done.